actors/market/src/lib.rs

```diff
@@ -788,15 +788,15 @@ impl Actor {
 
         let burn_amount = rt.transaction(|st: &mut State, rt| {
             // The sector deals mapping is removed all at once.
-            // Other deal clean-up is deferred to per-epoch cron.
+            // Note there may be some deal states that are not removed here,
+            // despite deletion of this mapping, e.g. for expired but not-yet-settled deals.
+            // The sector->deal mapping is no longer needed (the deal state has sector number too).
             let all_deal_ids = st.pop_sector_deal_ids(
```

Comment:
This line removes all provider-sector-deal mappings for the sectors, so the deleted code below was a no-op.